server/templates/includes/navbar.html

```diff
@@ -12,6 +12,11 @@
             <strong>Donate</strong>
           </a>
         </li>
+        <li class="c-navbar__item is-hidden-until-bp-l">
+          <a href="https://www.texastribune.org/about/" class="c-navbar__item-content c-navbar__clickable c-navbar__clickable--animated" ga-on="click" ga-event-category="navigation" ga-event-action="top nav click" ga-event-label="about us">
```

Comment:
**suggestion: (blocking)** Just for consistency and data tracking, could we add `?utm_medium=top-nav-about` to the URL?